procrastinate/utils.py

```diff
@@ -235,8 +235,12 @@ def _get_module_name(obj: Any) -> str:
 
 
 def get_full_path(obj: Any) -> str:
-
-    return f"{_get_module_name(obj)}.{obj.__name__}"
+    try:
+        return f"{_get_module_name(obj)}.{obj.__name__}"
+    except AttributeError as exc:
+        raise exceptions.FunctionPathError(
+            f"Couldn't automatically name {type(obj)}. Try passing `name=` to the task decorator"
```

Comment:
Maybe `Couldn't extract a relevant task name for {type(obj)}...`

Also, I wonder if printing `obj` itself would be better ?

Reply:
changed